### DISCUSSION TITLE
EV charger IP address in GivTCP

### DISCUSSION THREAD

**Discussion by boatieben** [BottlecapDave/HomeAssistant-OctopusEnergy#860] [Q&A]
Hello, the config page asks for an EV charger IP address, but there is none listed in my router device list. There is one item in the list with no IP listed, just a MAC address - which based on research seems to be in the same sort of range as others have identified. 

I suspect this field would not be there if it was not quite straightforward to source, which makes me think there is some kind of error in my configuration of the charger itself?

TIA!

**Comment by BottlecapDave** [comment]
Hello. As this question is related to a different integration, you might get a better response on either their github forums or on the HA community forums.
